{
    "original_source": {
        "database(url)": "https://www.cve.org",
        "software_system(CPE)": [
            "cpe:2.3:a:omniauth:omniauth:*:*:*:*:*:-:*:*"
        ],
        "fix_commit": "https://github.com/omniauth/omniauth/commit/43a396f181ef7d0ed2ec8291c939c95e3ed3ff00#diff-575abda9deb9b1a77bf534e898a923029b9a61e991d626db88dc6e8b34260aa2",
        "version_end_excluding": {
            "omniauth:omniauth": "1.9.2"
        },
        "path": {
            "file_path": ""
        }
    },
    "programming_language": "RUBY",
    "licences": {
        "database": "",
        "affected_repository": "MIT License"
    },
    "related_vulnerability_entry": {
        "CVE": [
            "CVE-2020-36599"
        ],
        "CWE": []
    }
}